src/fileio_func.h

```diff
@@ -149,4 +149,17 @@ static inline DIR *ttd_opendir(const char *path)
 	return opendir(OTTD2FS(path));
 }
 
+
+/** Auto-close a file upon scope exit. */
+class FileCloser {
+	FILE *_f;
```

Comment:
Our naming convention reserves the _ prefix for globals. I know that's wrong but should we maintain consistency?

Reply:
Probably yes.